### DISCUSSION TITLE
WatchYourLan lxc possible ?

### DISCUSSION THREAD

**Discussion by Mrdindon** [tteck/Proxmox#3775] [Ideas]
Would it be possible to make a script to deploy that ?
https://github.com/aceberg/WatchYourLAN

**Comment by towerhand** [comment]
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/watchyourlan.sh)"
`

**Reply by ptrsmk** [reply]
Thanks.

Now if only I could configure it properly.
